src/api/app/helpers/webui/request_helper.rb

```diff
@@ -198,4 +198,10 @@ def request_action_header(action, creator)
   def review_request_reason(bs_request, review)
     bs_request.request_history_elements.where(description_extension: review[:id]).pluck(:comment).first.presence || 'No reason given'
   end
+
+  def list_maintainers(maintainers)
+    maintainers.pluck(:login).map do |maintainer|
+      user_with_realname_and_icon(maintainer, no_icon: true, short: true)
```

Comment:
We should be consistent with the modal and also display icons. We do it too for other side links (so everything which is in the same list).

Reply:
Just updated this. Here is the view with icons:

![Screenshot from 2019-03-11 16-42-57](https://user-images.githubusercontent.com/22715037/54163444-b5202080-441e-11e9-96e1-62df33992dec.png)